stdlib/2/__builtin__.pyi

```diff
@@ -168,7 +168,7 @@ class int:
     def __rtruediv__(self, x: int) -> float: ...
     def __rmod__(self, x: int) -> int: ...
     def __rdivmod__(self, x: int) -> Tuple[int, int]: ...
-    def __pow__(self, x: int) -> Any: ...  # Return type can be int or float, depending on x.
+    def __pow__(self, x: int, modulo: Optional[int] = ...) -> Any: ...  # Return type can be int or float, depending on x.
```

Comment:
The arguments are actually positional-only

Reply:
Is there some special way to encode this?

Reply:
Start the names with two underscores, like `__modulo`.